src/runtime_src/core/pcie/driver/linux/xocl/subdev/firewall.c

```diff
@@ -292,7 +292,9 @@ static u32 check_firewall(struct platform_device *pdev, int *level)
 				xocl_ioaddr_to_baroff(xdev, res->start,
 					&bar_idx, &bar_off);
 			}
-			xocl_info(&pdev->dev, "AXI Firewall %d tripped, status: 0x%x, bar offset 0x%llx", i, val, bar_off);
+			/* To be improved: link IORESOURCE_MEM(0) with NODE_AF_BLP string */
+			xocl_info(&pdev->dev, "AXI Firewall %d tripped, status: 0x%x, bar offset 0x%llx, resource name:%s",
+			    i, val, bar_off, NODE_AF_BLP);
```

Comment:
the endpoint name is based on level. It should not use a fixed string here. And the resource name could be NULL for legacy shell.

Reply:
Thanks. I will make the changes.

Reply:
@houlz0507 would this version be better?